lib/plugins/aws/provider/awsProvider.js

```diff
@@ -123,6 +124,54 @@ const impl = {
   },
 };
 
+const baseAlbAuthorizerProperties = {
+  onUnauthenticatedRequest: { enum: ['allow', 'authenticate', 'deny'] },
+  requestExtraParams: {
+    type: 'object',
+    maxProperties: 10,
+    additionalProperties: { type: 'string' },
+  },
+  scope: { type: 'string' },
+  sessionCookieName: { type: 'string' },
+  sessionTimeout: { type: 'integer' },
+};
+
+const oidcAlbAuthorizer = {
+  type: 'object',
+  properties: {
+    type: { const: 'oidc' },
+    authorizationEndpoint: { format: 'uri' },
+    clientId: { type: 'string' },
+    clientSecret: { type: 'string' },
+    issuer: { format: 'uri' },
+    tokenEndpoint: { format: 'uri' },
+    userInfoEndpoint: { format: 'uri' },
+    ...baseAlbAuthorizerProperties,
+  },
+  required: [
+    'type',
+    'authorizationEndpoint',
+    'clientId',
+    'issuer',
+    'tokenEndpoint',
+    'userInfoEndpoint',
+  ],
+  additionalProperties: false,
+};
+
+const cognitoAlbAuthorizer = {
+  type: 'object',
+  properties: {
+    type: { const: 'cognito' },
+    userPoolArn: { $ref: '#/definitions/awsArnString' },
```

Comment:
This seems not addressed https://github.com/serverless/serverless/pull/8291#discussion_r495842300

Reply:
Sorry about all this back&forth, I keep forgetting to unfold the hidden comments so I keep missing a few.